src/ui.rs

```diff
@@ -141,47 +226,130 @@ impl<'a> State<'a> {
     }
 }
 
+/// Process SDL2 events while getting number of times to repeat action
+pub fn process_multi_normal_mode<'a>(state: &mut State<'a>, event: Event) -> MultiNormalAction<'a> {
+    use sdl2::event::WindowEvent::*;
+    use sdl2::keyboard::Keycode::*;
+
+    match event {
+        Event::Quit { .. } => MultiNormalAction::Quit,
+
+        Event::TextInput { text, .. } => match text.as_str() {
+            // Number of times to repeat operation
+            "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9" => {
+                let previous_count = state.repeat;
+                // Safe to unwrap as only digits were matched
+                let next_digit = text.parse::<usize>().unwrap();
```

Comment:
Why parse and add it on here? Instead of just storing it in a `String` or maybe `Cow` borrowed string and then when the user hits an action then parse it? This would also make it so you don't have to divide by 10 in `mod.rs` to truncate a digit

Reply:
Parsing immediately was one way to reduce code duplication when the text is not `0-9`.
The amount of times would not have to be parsed on every action. 
I think I could move the parsing out of the `ui` event -> MultiNormalAction translation if Normal mode read from the same field `repeat`.